core/storage/topic/gae_models.py

```diff
@@ -165,6 +167,27 @@ class TopicSummaryModel(base_models.BaseModel):
     version = ndb.IntegerProperty(required=True)
 
 
+class SubtopicPageModel(base_models.BaseModel):
+    """Subtopic model for an Oppia Topic.
+
+    This stores the html data for the subtopic landing page for all subtopics
```

Comment:
This stores the HTML data for a subtopic page.

(Omit the rest; otherwise it sounds like one model instance stores the data for all subtopics in Oppia.)

Reply:
Done.